### PR TITLE
Updated Cache-Control documentation page

### PR DESCRIPTION
**Pull request by Secreto31126**
https://developer.mozilla.org/es/docs/Web/HTTP/Headers/Cache-Control#request_directives

### Description

I noticed there was a missing # in the no-store subtitle.

### Motivation

I'm doing it because I have free time :)

### Additional details

There's something weird going on in ##casos-de-uso, the title is duplicated, but I didn't want to touch it. Also, I'm quite sure the line 255 shouldn't be translated.

### Related issues and pull requests

N/A

### CODE REVIEW

**Review by Graywolf9 — COMMENTED** [review]
Thank you so much for your time and help!

As you say I noticed that there is a lot of mistakes in this article from bad format titles to outdated info

Do you want to update the whole document? Don't worry about time o errors we are here to help you in all you need. We take `en-US` as reference for translate: https://developer.mozilla.org/en-US/docs/Web/HTTP/Headers/Cache-Control and if you want you can take that source code to ensure the correct format and only translate the text: https://github.com/mdn/content/blob/main/files/en-us/web/http/headers/cache-control/index.md?plain=1

Let me know what do you think, we can work in this PR or another one.

P.S.: If your more confortable we can talk in spanish

**Comment by Secreto31126** [comment]
Sure, I can do it. I will mark the PR as draft and reopen it with a comment when it's ready.

**Comment by Secreto31126** [comment]
Quick question, [Google's Web Light](https://developers.google.com/search/docs/advanced/mobile/web-light?visit_id=637855965115455923-776951611&rd=1) seems to have been retired less than a month ago. [It's mentioned here](https://github.com/mdn/content/blob/65d9812941cd4e94334c6b8732e3c5e5da3ecd90/files/en-us/web/http/headers/cache-control/index.md?plain=1#L216).

What should I do? Leave it as is?

**Comment by Secreto31126** [comment]
I think it's ready, let me know what you think

File: files/es/web/http/headers/cache-control/index.md [hunk 1]
```diff
@@ -63,93 +63,104 @@ Las directivas `Cache-Control` estándar están definidas a continuación.
...
+  - : Implementación que mantiene peticiones y respuestas para reusarlas en peticiones posteriores. Puede ser tanto una cache privada como una compartida.
 - `Caché compartida`
   - : Caché existente entre el servidor de origen y los clientes (p. ej. Proxy, CDN). Almacena una sola respuesta para reutilizarla en multiples usuarios — por tanto los desarrolladores deberían evitar que el almacenamiento de contenidos personalizados sea cacheado en la caché compartida.
 - `Caché privada`
-  - : Caché existente en el cliente. También conocida como _caché local_, o _caché del navegador_, etc. Puede almacenar y reutilizar contenido personalizado para un único usuario.
+  - : Caché existente en el cliente. También conocida como _caché local_, o _caché del navegador_. Puede almacenar y reutilizar contenido personalizado para un único usuario.
 - `Respuesta almacenada`
-  - : Almacena una respuesta en caches cuando es cacheable. Pero no siempre es reutilizada en ese momento. (Normalmente "caché" significa almacenar una respuesta).
+  - : Almacena una respuesta en caches cuando es cacheable. Pero no siempre es reutilizada tal cual. (Normalmente "caché" significa almacenar una respuesta).
```
Comment (Graywolf9) [inline files/es/web/http/headers/cache-control/index.md:79]:
```suggestion
  - : Almacena una respuesta en caches cuando es _cacheable_. Pero no siempre es reutilizada tal cual. (Normalmente "caché" significa almacenar una respuesta).
```

File: files/es/web/http/headers/cache-control/index.md [hunk 2]
```diff
@@ -63,93 +63,104 @@ Las directivas `Cache-Control` estándar están definidas a continuación.
...
 - `Respuesta reciente`
-  - : Indica que una respuesa es reciente. Esto normalmente significa que la respuesta puede ser reutilizada para las subsiguientes peticiones, dependiendo de las directivas de petición.
+  - : Indica que una respuesa es reciente ([fresh](/es/docs/Web/HTTP/Caching#estados_fresh_y_stale_según_la_edad)). Esto normalmente significa que la respuesta puede ser reutilizada para las subsiguientes peticiones, dependiendo de las directivas de petición.
 - `Respuesta pasada`
-  - : Indica que la respuestá está pasada. Normalmente significa que la respuesta ya no puede ser reutilizada. El almacenamiento caché no requiere que las respuestas pasadas sean eliminada inmediatamente, por que la revalidación puede cambiar la respuesta de pasada a reciente de nuevo.
+  - : Indica que la respuestá está pasada ([stale](/es/docs/Web/HTTP/Caching#estados_fresh_y_stale_según_la_edad)). Normalmente significa que la respuesta ya no puede ser reutilizada. El almacenamiento caché no requiere que las respuestas pasadas sean eliminada inmediatamente, por que la revalidación puede cambiar la respuesta de pasada a reciente de nuevo.
```
Comment (Graywolf9) [inline files/es/web/http/headers/cache-control/index.md:87]:
```suggestion
  - : Indica que la respuestá está obsoleta ([stale](/es/docs/Web/HTTP/Caching#estados_fresh_y_stale_según_la_edad)). Normalmente significa que la respuesta ya no puede ser reutilizada. El almacenamiento caché no requiere que las respuestas pasadas sean eliminada inmediatamente, por que la revalidación puede cambiar la respuesta de pasada a reciente de nuevo.
```

File: files/es/web/http/headers/cache-control/index.md [hunk 3]
```diff
@@ -63,93 +63,104 @@ Las directivas `Cache-Control` estándar están definidas a continuación.
...
 - `Respuesta reutilizada`
-  - : Reutiliza respuestas cacheadas para las subsiguientes peticiones,
+  - : Reutiliza respuestas cacheadas para las subsiguientes peticiones.
 - `Revalidar respuesta`
-  - : Pregunta al servidor de origen si una respuesta almacenada sigue siendo reciente o no. Normalmente se realiza a través de una petición condicionada.
+  - : Pregunta al servidor de origen si una respuesta almacenada sigue siendo reciente o no ([fresh o stale](/es/docs/Web/HTTP/Caching#estados_fresh_y_stale_según_la_edad)). Normalmente se realiza a través de una petición condicionada.
 - `Respuesta reciente`
-  - : Indica que una respuesa es reciente. Esto normalmente significa que la respuesta puede ser reutilizada para las subsiguientes peticiones, dependiendo de las directivas de petición.
+  - : Indica que una respuesa es reciente ([fresh](/es/docs/Web/HTTP/Caching#estados_fresh_y_stale_según_la_edad)). Esto normalmente significa que la respuesta puede ser reutilizada para las subsiguientes peticiones, dependiendo de las directivas de petición.
 - `Respuesta pasada`
```
Comment (Graywolf9) [inline files/es/web/http/headers/cache-control/index.md:86]:
```suggestion
- `Respuesta obsoleta`
```

File: files/es/web/http/headers/cache-control/index.md [hunk 4]
```diff
@@ -63,93 +63,104 @@ Las directivas `Cache-Control` estándar están definidas a continuación.
...
-  - : Indica que la respuestá está pasada. Normalmente significa que la respuesta ya no puede ser reutilizada. El almacenamiento caché no requiere que las respuestas pasadas sean eliminada inmediatamente, por que la revalidación puede cambiar la respuesta de pasada a reciente de nuevo.
+  - : Indica que la respuestá está pasada ([stale](/es/docs/Web/HTTP/Caching#estados_fresh_y_stale_según_la_edad)). Normalmente significa que la respuesta ya no puede ser reutilizada. El almacenamiento caché no requiere que las respuestas pasadas sean eliminada inmediatamente, por que la revalidación puede cambiar la respuesta de pasada a reciente de nuevo.
```
Comment (Graywolf9) [inline files/es/web/http/headers/cache-control/index.md:87]:
```suggestion
  - : Indica que la respuestá está obsoleta ([stale](/es/docs/Web/HTTP/Caching#estados_fresh_y_stale_según_la_edad)). Normalmente significa que la respuesta ya no puede ser reutilizada. El almacenamiento caché no requiere que las respuestas obsoletas sean eliminadas inmediatamente, por que la revalidación puede cambiar la respuesta de obsoleta a reciente de nuevo.
```

File: files/es/web/http/headers/cache-control/index.md [hunk 5]
```diff
@@ -179,119 +190,127 @@ Si una petición no tiene la cabecera `Authorization`, o si ya estás usando `s-
...
 ```
 
 `max-age=0` es una solución alternativa para `no-cache`, porque muchas implementaciones de caché antiguas (HTTP/1.0) no son compatibles con `no-cache`. Los navegadores más recientes siguen usando `max-age=0` en "recargas" (por compatibilidad con versiones anteriores) y, alternativamente, usan `no-cache` para provocar una "recarga forzada".
 
-### `max-stale`
+Si el valor de `max-age` no es positivo (por ejemplo, `-1`) o no es entero (por ejemplo, `3599.99`), el comportamiento del cache es indefinido. Sin embargo, la sección [Calculating Freshness Lifetime](https://httpwg.org/specs/rfc9111.html#calculating.freshness.lifetime) de las especificaciones HTTP establece:
+
+> Caches are encouraged to consider responses that have invalid freshness information to be stale.
+
+> Traducción: Se recomienda a los caches considerar una respuesta como antigua si la información de edad es inválida.
```
Comment (Graywolf9) [inline files/es/web/http/headers/cache-control/index.md:305]: To keep a discreete format
```suggestion
Traducción: Se recomienda a las caches considerar una respuesta como antigua si la información de edad es inválida.
```

File: files/es/web/http/headers/cache-control/index.md [hunk 6]
```diff
@@ -443,16 +444,12 @@ Alternativamente, `Clear-Site-Data` puede borrar la memoria caché del navegador
...
 {{Compat}}
 
-[HTTP Caching](https://httpwg.org/http-core/draft-ietf-httpbis-cache-latest.html)
-[RFC5861 - HTTP Cache-Control Extensions for Stale Content](https://datatracker.ietf.org/doc/html/rfc5861)
-[RFC8246 - HTTP Immutable Responses](https://datatracker.ietf.org/doc/html/rfc8246)
-
 ## Véase también
 
-- [HTTP Caching FAQ](/es/docs/Web/HTTP/Caching)
+- [HTTP caching](/es/docs/Web/HTTP/Caching)
```
Comment (Graywolf9) [inline files/es/web/http/headers/cache-control/index.md:449]:
```suggestion
- [Caché en HTTP](/es/docs/Web/HTTP/Caching)
```

File: files/es/web/http/headers/cache-control/index.md [hunk 7]
```diff
@@ -443,16 +444,12 @@ Alternativamente, `Clear-Site-Data` puede borrar la memoria caché del navegador
...
 - [Caching Tutorial for Web Authors and Webmasters](https://www.mnot.net/cache_docs/)
 - [Caching best practices & max-age gotchas](https://jakearchibald.com/2016/caching-best-practices/)
 - [Cache-Control for Civilians](https://csswizardry.com/2019/03/cache-control-for-civilians/)
```
Comment (Graywolf9) [inline files/es/web/http/headers/cache-control/index.md:452]:
```suggestion
- [Tutorial de almacenamiento en caché para autores web y webmasters](https://www.mnot.net/cache_docs/)
- [Prácticas recomendadas de almacenamiento en caché y trucos de edad máxima](https://jakearchibald.com/2016/caching-best-practices/)
- [Control de caché para civiles](https://csswizardry.com/2019/03/cache-control-for-civilians/)
```

File: files/es/web/http/headers/cache-control/index.md [hunk 8]
```diff
@@ -443,16 +444,12 @@ Alternativamente, `Clear-Site-Data` puede borrar la memoria caché del navegador
...
-- {{HTTPHeader("Age")}}
-- {{HTTPHeader("Expires")}}
-- {{HTTPHeader("Pragma")}}
+- [RFC 9111 – HTTP Caching](https://httpwg.org/specs/rfc9111.html)
+- [RFC 5861 – HTTP Cache-Control Extensions for Stale Content](https://httpwg.org/specs/rfc5861.html)
+- [RFC 8246 – HTTP Immutable Responses](https://httpwg.org/specs/rfc8246.html)
```
Comment (Graywolf9) [inline files/es/web/http/headers/cache-control/index.md:455]:
```suggestion
- [RFC 9111 – Caché en HTTP](https://httpwg.org/specs/rfc9111.html)
- [RFC 5861 – Extensiones de control de caché HTTP para contenido obsoleto](https://httpwg.org/specs/rfc5861.html)
- [RFC 8246 – Respuestas HTTP inmutables](https://httpwg.org/specs/rfc8246.html)
```

**Review by Graywolf9 — APPROVED** [review]
thank you so much for your work, about "Google's Web Light" the way is to make a PR to modify in `en-US` once approved make the change here

**Pull request merged**